### PR TITLE
housekeeping:

### PR DESCRIPTION
**Pull request by oyvindberg**
- remove most unused warnings
- introduce `bleep.Discard` to do so
- bump bleep
- optimize imports
- fmt
- enforce `.model` prefix

### CODE REVIEW

**Comment by oyvindberg** [comment]
there is a mountain of unused warnings for xml literals for scala 3 still, doesn't seem fixable unless rewritten into something else.

**Comment by KristianAN** [comment]
yes, the warnings for xml literals is something we also have a lot of at $work. I have not found a workaround yet, and there does not seem to be much activity to fix it.

**Comment by oyvindberg** [comment]
yeah, also couldn't find anything except some old issues. I think the plan for the last decade has been to remove the xml literals and replace with string interpolators which nobody has written yet

File: bleep-cli/src/scala/bleep/commands/InstallZshTabCompletions.scala [hunk 1]
```diff
@@ -32,7 +32,7 @@ case class InstallZshTabCompletions(userPaths: UserPaths, logger: Logger, stdout
       println(completionScript)
     } else {
       logger.info(s"Writing $completionScriptDest")
-      FileSync.softWriteBytes(completionScriptDest, completionScript.getBytes(StandardCharsets.UTF_8)): Unit
+      FileSync.softWriteBytes(completionScriptDest, completionScript.getBytes(StandardCharsets.UTF_8)).discard()
```
Comment (sideeffffect) [inline bleep-cli/src/scala/bleep/commands/InstallZshTabCompletions.scala:35]: Why not just
```scala
val _ = FileSync.softWriteBytes(completionScriptDest, completionScript.getBytes(StandardCharsets.UTF_8))
```
There's no need to invent custom extension methods to do a simple thing like this...

Comment (oyvindberg) [reply bleep-cli/src/scala/bleep/commands/InstallZshTabCompletions.scala:35]: Believe it or not, cross compiling between strict 2.13 and 3 provides no way to resolve all unused cases. And you need to use 3-4-5 different syntaxes to resolve most unused cases. It's laughable, really. Inventing this is way easier to deal with

Comment (sideeffffect) [reply bleep-cli/src/scala/bleep/commands/InstallZshTabCompletions.scala:35]: Oh boy :scream_cat: fair enough then...

**Pull request merged**